pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -160,15 +233,38 @@ func valueString(v *buildexpression.Value) string {
 }
 
 func apString(f *buildexpression.Ap) string {
+	if f.Name == reqFuncName {
+		return apReqString(f)
+	}
+
 	buf := bytes.Buffer{}
 	buf.WriteString(fmt.Sprintf("%s(\n", f.Name))
+
 	for i, argument := range f.Arguments {
 		buf.WriteString(indent(valueString(argument)))
+
 		if i+1 < len(f.Arguments) {
 			buf.WriteString(",")
 		}
+
 		buf.WriteString("\n")
 	}
+
+	buf.WriteString(")")
+	return buf.String()
+}
+
+func apReqString(f *buildexpression.Ap) string {
+	buf := bytes.Buffer{}
+	buf.WriteString(fmt.Sprintf("%s(", f.Name))
+	for i, argument := range f.Arguments {
+		buf.WriteString(strutils.RemoveSpaces(valueString(argument)))
+
+		if i+1 < len(f.Arguments) {
+			buf.WriteString(", ")
+		}
+	}
```

Comment:
As mentioned previously, this is a lot of extra effort to remove spaces. Let's just keep them and simplify the code for now.